exercises/solutions/resources/resource_06_12.py

```diff
@@ -211,28 +204,18 @@ def __init__(self, page):
             "[data-test=Breadcrumbs-step-PASSENGER] [aria-current=step]"
         )
 
-    def wait_for_available_connections_to_be_displayed(self):
-        self.result_card_wrapper.wait_for()
-
     def check_a_transport_option_checkbox(self, option: str = None):
         self.page.locator(f"[class*=FilterWrapper]:has([data-test=TransportOptionCheckbox-{option.lower()}])").click()
-        self.wait_for_search_result_page_to_be_reloaded()
-
-    def wait_for_search_result_page_to_be_reloaded(self):
```

Comment:
Hopefully it stays stable without this wait 🤞

Reply:
Pls, in case it is stable, remove this row from the corresponding `exercises/tasks/resources/resource_06_12.py` as well 🙏